app/helpers/jobs_helper.rb

```diff
@@ -0,0 +1,2 @@
+module JobsHelper
```

Comment:
I left it there because there are other empty helpers left in `app/helpers/`. I might just put up a patch removing them all. See PRl#183.